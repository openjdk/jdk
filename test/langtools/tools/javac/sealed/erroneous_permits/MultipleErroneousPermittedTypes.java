/*
 * @test /nodynamiccopyright/
 * @bug 8347562
 * @summary Erroneous types in permit clause shouldn't cause more errors/crashes
 * @compile/fail/ref=MultipleErroneousPermittedTypes.out -XDrawDiagnostics MultipleErroneousPermittedTypes.java
 */
sealed class Permits<X, Y> permits X, Y {}
