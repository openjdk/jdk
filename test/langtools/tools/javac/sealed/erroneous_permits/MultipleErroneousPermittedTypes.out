MultipleErroneousPermittedTypes.java:7:36: compiler.err.illegal.forward.ref
MultipleErroneousPermittedTypes.java:7:39: compiler.err.illegal.forward.ref
2 errors
